Riot/Managers/LocalAuthentication/LocalAuthenticationService.swift

```diff
@@ -44,6 +44,11 @@ class LocalAuthenticationService: NSObject {
         NotificationCenter.default.addObserver(self, selector: #selector(applicationWillResignActive), name: UIApplication.willResignActiveNotification, object: nil)
     }
     
+    /// Whether the application currently showing the biometrics setup or unlock dialog.
+    /// Showing biometrics dialog will cause the app to resign active.
+    /// This property can be used in order to distinguish real resignations and biometrics case.
+    static var isShowingBiometrics: Bool = false
```

Comment:
Instead of having this boolean here I think we can create a `BiometricsVerificationPresenter`

```
enum BiometricsVerificationPresenterError: Error {
    case unknown
}

final class BiometricsVerificationPresenter {

    // Use an instance var instead of static one when BiometricsVerificationPresenter will be managed by AppCoordinator for example
    static private(set) var isShowingBiometrics: Bool = false

    func present(with message: String, completion: @escaping (Result<Void, Error>) -> Void) {
        
        // We can also imagine to not present is there is currently biometrics presented
        BiometricsVerificationPresenter.isShowingBiometrics = true

        LAContext().evaluatePolicy(.deviceOwnerAuthentication, localizedReason: message) { (success, error) in
            if success {
                // Complete after a little delay
                DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) {
                    BiometricsVerificationPresenter.isShowingBiometrics = false
                    completion(.success(Void()))
                }
            } else {
                let finalError: Error
                
                if let error = error {
                    finalError = error
                } else {
                    finalError = BiometricsVerificationPresenterError.unknown
                }
                
                BiometricsVerificationPresenter.isShowingBiometrics = false
                completion(.failure(finalError))
            }
        }
    }
}
```

Reply:
Updated